rotkehlchen/api/v1/fields.py

```diff
@@ -199,9 +199,6 @@ def _deserialize(
 
 class TimestampMSField(fields.Field):
 
-    def __init__(self, **kwargs: Any) -> None:
-        super().__init__(**kwargs)
-
```

Comment:
No idea why pylint finds this now ... wasn't found before when pylint did its own folder traversing